soap.go

```diff
@@ -4,6 +4,7 @@ import (
 	"bytes"
 	"encoding/xml"
 	"fmt"
+	"github.com/pkg/errors"
```

Comment:
why do not use golang errors pkg?
```suggestion
	"errors"
```

Reply:
Good one @tiaguinho. I've applied the suggestion.